### DISCUSSION TITLE
I think it might be time to display osu!lazer as the main option on the download page.

### DISCUSSION THREAD

**Discussion by OwenCMYK** [ppy/osu#27452] [Ideas]
Obviously this will need some discussion from big community members, but I just wanted to like... kickstart that process if possible. I've been seeing a lot of posts on the Reddit asking "which version should I get?" and the overwhelming majority of people always suggest lazer.

Obviously Flyte (or whoever this fall under) should probably do their magic to redesign this, but here's a mockup of what I think the download screen could roughly look like:
![try osu!lazer](https://github.com/ppy/osu/assets/42657486/0dc42d5d-fc8b-42f4-8808-d79c7665b1a6)

Apologies for the very scuffed look, I designed this very quickly in less-than-professional software just to vaguely resemble the original site. Feel free to weigh in with your opinions.

**Comment by vatei** [comment]
Not yet imo because some important features from stable are still missing including better skin management, performance, medals (although they seem to be coming soon), complete beatmap editor, and lots of gameplay choices remain to be discussed and implemented like enabling pp on CL and other mods. And I'm only talking about std, apparently mania and maybe other gamemodes fall a bit short of expectations too.
Not to disregard the pile of work the team already did of course, but lots more remain before Lazer can be a proper replacement

**Reply by OwenCMYK** [reply]
Ah, I was not aware that the other rulesets still had larger issues, that would definitely be a contributing factor (along with metals which I kinda forgot about lol). But I'm pretty sure performance for on most devices is better than stable. And I think skin management is already better than stable (stable doesn't even let you delete skins without editing game files manually). CL mod is kinda in a grey area because primarily it's only for older folks who've grown used to stable, so for new players that won't have much of an effect, but I can definitely see it being an issue.

**Reply by vatei** [reply]
About skin management, on the flip side you can't edit the skin files at all from the game yet, and the component system doesn't cover nearly as much as skin.ini 
For performance, "works on my machine" I guess, but there are still lots of reports of stutters in gameplay. And the UI is definitely less snappy than stable. Although Evast has been working hard on those allocations so there may be some improvements soon !
It's not only CL, DT adjust isn't ranked, and mod multipliers aren't final anyway

**Comment by bdach** [comment]
I don't think this would be justified given the current user numbers and the high priority issues the game still has that need addressing.

**Reply by OwenCMYK** [reply]
Oki
